lib/Github/Api/Notification.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Github\Api;
+
+use DateTime;
+
+/**
+ * API for accessing Notifications from your Git/Github repositories.
+ *
+ * Important! You have to be authenticated to perform these methods
+ *
+ * @link   https://developer.github.com/v3/activity/notifications/
+ * @author Dennis de Greef <github@link0.net>
+ */
+class Notification extends AbstractApi
+{
+    /**
+     * Get a listing of a notifications
+     * @link https://developer.github.com/v3/activity/notifications/
+     *
+     * @param bool $includingRead
+     * @param bool $participating
+     * @param null $since
```

Comment:
By the way, in `markRead()` you treat `$since` as `DateTime`. Might be a good idea to to that here as well.